src/components/molecules/Login.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { ActionButton } from "../atoms/ActionButton";
+
+/**
+ * This component is for the login button :)
+ */
+export function Login(props) {
```

Comment:
this component is really not needed in my opinion just use ActionButton component in header component and it will be available everywhere

Reply:
agree is mostly for future use (next sprint) box and text from the design